# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fullscreen/api/element-request-fullscreen-timing.html.ini
# Commit: 425cbe3c033b
# Full Hash: 425cbe3c033bfc25795979513ebd2bb9fda7338d
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - [wpt-sync] Update web-platform-tests to 2d5d222739fdf91743f8cbc58094976a613194e7, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 2d5d222739fdf91743f8cbc58094976a613194e7
# ==============================================================================

diff -r 732bcd7ac971 -r 425cbe3c033b testing/web-platform/meta/fullscreen/api/element-request-fullscreen-timing.html.ini
--- a/testing/web-platform/meta/fullscreen/api/element-request-fullscreen-timing.html.ini	Wed Nov 27 22:05:28 2024 +0000
+++ b/testing/web-platform/meta/fullscreen/api/element-request-fullscreen-timing.html.ini	Wed Nov 27 22:05:56 2024 +0000
@@ -3,14 +3,13 @@
     if (os == "mac") and not debug: [OK, TIMEOUT]
   [Timing of fullscreenchange and resize events]
     expected:
-      if (os == "linux") and fission and tsan: FAIL
-      if (os == "linux") and not fission and debug: [PASS, FAIL]
-      if (os == "linux") and not fission and not debug: FAIL
+      if (os == "linux") and not tsan and not fission and not debug: PASS
+      if (os == "mac") and debug: [FAIL, PASS]
       if (os == "mac") and not debug: [FAIL, PASS, TIMEOUT]
+      if (os == "linux") and tsan: PASS
       [PASS, FAIL]
 
   [Timing of fullscreenerror event]
     expected:
-      if (os == "linux") and debug and fission: [FAIL, PASS]
       if (os == "win") and not debug and (processor == "x86_64"): PASS
       [PASS, FAIL]